src/main/java/com/jvm_bloggers/core/social/fb/publisher/RestFbPublisher.java

```diff
@@ -0,0 +1,41 @@
+package com.jvm_bloggers.core.social.fb.publisher;
+
+import com.jvm_bloggers.entities.fb.FacebookPost;
+import com.restfb.FacebookClient;
+import com.restfb.Parameter;
+import com.restfb.types.FacebookType;
+import com.restfb.types.Post;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+
+import static com.jvm_bloggers.ApplicationProfiles.PRODUCTION;
+
+@Slf4j
+@Component
+@Profile({PRODUCTION})
+class RestFbPublisher {
```

Comment:
Maybe I'm missing something but how it can be an implementation of a `FacebookPublisher` w/o `implements FacebookPublisher` clause?